src/snowflake/connector/cursor.py

```diff
@@ -929,8 +929,8 @@ def execute(
 
         if _do_reset:
             self.reset()
-        command = command.strip(" \t\n\r") if command else None
```

Comment:
Change `None` to "", otherwise the following log formatters will fail.

Reply:
that makes sense.